<launch>
  <arg name="camera_name"	default="a_phoxi_m_camera"/>
  <!-- <arg name="id" default="InstalledExamples-PhoXi-example(File3DCamera)"/> -->
  <!-- <arg name="id" default="InstalledExamples-basic-example"/> -->
  <!-- <arg name="id"		default="1711015"/> -->
  <arg name="id"		default="2018-03-027-LC3"/>
  <arg name="trigger_mode"	default="1"/>
  <arg name="intensity_scale"	default="0.5"/>
  <arg name="frame"		default="camera_link_from_b_bot_estimated"/>
  <node name="$(arg camera_name)"
	pkg="o2as_phoxi_camera" type="o2as_phoxi_camera" output="screen">
    <param name="id"			value="$(arg id)"/>
    <param name="trigger_mode"		value="$(arg trigger_mode)"/>
    <param name="intensity_scale"	value="$(arg intensity_scale)"/>
    <param name="frame"			value="$(arg frame)"/>
  </node>
</launch>
